crates/storage/trie/node/branch.rs

```diff
@@ -317,8 +321,18 @@ mod test {
     use ethereum_types::H256;
 
     use super::*;
+    use crate::trie::libdbmx_db::LibmdbxTrieDb;
     use crate::{pmt_node, trie::Trie};
 
+    #[cfg(test)]
+    /// Creates a new trie based on a temporary DB
+    fn new_temp() -> Trie<LibmdbxTrieDb> {
+        Trie {
+            state: TrieState::new(LibmdbxTrieDb::init_temp()),
+            root: None,
+        }
+    }
+
```

Comment:
We already have the same method in the trie test module, maybe we should move it to `test_utils` instead of copying it